### ISSUE TITLE
Cannot send a new extension for review

### ISSUE THREAD

**Issue by mauromascarenhas** [mozilla/addons#315]
### Describe the problem and steps to reproduce it:
(Please include as many details as possible.)

As you can see problem in the attached image, after follow the steps below, I just get error messages from the AMO.

1. Navigate to : https://addons.mozilla.org/pt-BR/developers/addon/submit/distribution
2. Choose "On this site" option and click in "Continue" (I don't know how is it exactly in english, because mine is in pt_BR).
3. Select the attached file (SafeSearch-Enforcer.zip) and the addon is validaded successfully.
4. I am able to fill all the blanks, but cannot see the name and the description of the extension.
5. Click in "Continue" and get the error messages like in Screenshot #1.

### What happened?

As you can see problem in the attached image, I just get wrong error messages from the AMO.
But after that, I realized that there are many submissions of this plugin for review (Screenshot #3 - there are more activities logged because I have tried to send it several times, but in none of them I succeeded).

### What did you expect to happen?

I strongly believe that just a success message should appear after that, not those errors.

### Anything else we should know?

Attachments:

* Addon : ![SafeSearch-Enforcer.zip](https://github.com/mozilla/addons/files/721686/SafeSearch-Enforcer.zip)

* Screenshot #1: ![bug](https://cloud.githubusercontent.com/assets/7539096/22179989/009ccb6e-e042-11e6-98ee-b59c89f46392.png)

* Screenshot #2: ![bug](https://cloud.githubusercontent.com/assets/7539096/22179979/68edf126-e041-11e6-9676-88cc390cd982.png)

* Screenshot #3: ![bug](https://cloud.githubusercontent.com/assets/7539096/22180011/8f071634-e042-11e6-911a-d147f5ef5fb2.png)

**Comment by andymckay** [comment]
Which locale is your browser set to @mauromascarenhas, is it pt? I note that pt is one of the localisations in the _locales folder of your extension.

**Comment by mauromascarenhas** [comment]
Yes, it is set to pt_BR. Furthermore the translation is read by the browser on "about:addons" page with portuguese strings.
